src/main/java/edu/kis/powp/jobs2d/command/ICompoundCommand.java

```diff
@@ -9,4 +11,8 @@ public interface ICompoundCommand extends DriverCommand {
 
 	public Iterator<DriverCommand> iterator();
 
+	@Override
+	default Object copyDriverCommand() throws CloneNotSupportedException {
+		return DriverCommand.super.copyDriverCommand();
+	}
```

Comment:
This default method can use ComplexCommand and iterator() to implement recursive (deep) copy.